### PR TITLE
Added translation of 3 files as mentioned in issue

### PR DESCRIPTION
**Pull request by aneeshd27**
Worked on Issue- #1771
Added translation of quick-start.md,installation.md and index.md and made changes in yml file.
@Wauplin can you please take a look at it ?
It would mean to lot.
Thank you.

### CODE REVIEW

File: docs/source/hi/installation.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+```bash
+pip install --upgrade huggingface_hub
+```
+
+एक बार हो जाने पर, [चेक इंस्टालेशन](#चेक-इंस्टॉलेशन) सही ढंग से काम कर रहा है।
+
+### वैकल्पिक निर्भरताएँ स्थापित करें
+
+`huggingface_hub` की कुछ निर्भरताएं [वैकल्पिक](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html# optional-dependcies) हैं क्योंकि उन्हें `huggingface_hub` की मुख्य विशेषताओं को चलाने की आवश्यकता नहीं है। हालाँकि, यदि वैकल्पिक निर्भरताएँ स्थापित नहीं हैं तो `huggingface_hub` की कुछ सुविधाएँ उपलब्ध नहीं हो सकती हैं।
```
Comment (Vaibhavs10) [inline docs/source/hi/installation.md:45]:
```suggestion
`huggingface_hub` की कुछ निर्भरताएं [वैकल्पिक](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies) हैं क्योंकि उन्हें `huggingface_hub` की मुख्य विशेषताओं को चलाने की आवश्यकता नहीं है। हालाँकि, यदि वैकल्पिक निर्भरताएँ स्थापित नहीं हैं तो `huggingface_hub` की कुछ सुविधाएँ उपलब्ध नहीं हो सकती हैं।
```

File: docs/source/hi/installation.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- हब पर फ़ाइलपथ में विशेष वर्ण हो सकते हैं (जैसे `"path/to?/my/file"`)। विंडोज़ है
+[विशेष वर्णों] पर अधिक प्रतिबंधात्मक(https://learn.microsoft.com/en-us/windows/win32/intl/character-sets-used-in-file-names)
+जिससे विंडोज़ पर उन फ़ाइलों को डाउनलोड करना असंभव हो जाता है। उम्मीद है कि यह एक दुर्लभ मामला है.
+यदि आपको लगता है कि यह कोई गलती है तो कृपया रेपो मालिक से संपर्क करें या इसका पता लगाने के लिए हमसे संपर्क करें
+एक समाधान।
+
+
+## अगले कदम
+
+एक बार जब `हगिंगफेस_हब` आपकी मशीन पर ठीक से स्थापित हो जाए, तो आप चाहेंगे
```
Comment (Vaibhavs10) [inline docs/source/hi/installation.md:158]:
```suggestion
एक बार जब `huggingface_hub` आपकी मशीन पर ठीक से स्थापित हो जाए, तो आप चाहेंगे
```

File: docs/source/hi/installation.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+## विंडोज़ सीमाएँ
+
+हर जगह अच्छे एमएल का लोकतंत्रीकरण करने के अपने लक्ष्य के साथ, हमने `हगिंगफेस_हब` का निर्माण किया
+क्रॉस-प्लेटफ़ॉर्म लाइब्रेरी और विशेष रूप से यूनिक्स-आधारित और विंडोज़ दोनों पर सही ढंग से काम करने के लिए
+सिस्टम. हालाँकि, ऐसे कुछ मामले हैं जहाँ `huggingface_hub` की कुछ सीमाएँ हैं
+विंडोज़ पर चलाएँ. यहां ज्ञात मुद्दों की एक विस्तृत सूची दी गई है। कृपया हमें बताएं यदि आप
+[जीथब पर एक मुद्दा] (https://github.com/huggingface/huggingface_hub/issues/new/choose) खोलकर किसी भी अज्ञात समस्या का सामना करें।
+
+- `हगिंगफेस_हब` का कैश सिस्टम डाउनलोड की गई फ़ाइलों को कुशलतापूर्वक कैश करने के लिए सिम्लिंक पर निर्भर करता है
```
Comment (Vaibhavs10) [inline docs/source/hi/installation.md:145]:
```suggestion
- `huggingface_hub` का कैश सिस्टम डाउनलोड की गई फ़ाइलों को कुशलतापूर्वक कैश करने के लिए सिम्लिंक पर निर्भर करता है
```

**Review by Vaibhavs10 — COMMENTED** [review]
Very cool! Thanks for beginning translation work! <3

There are some inconsistencies in the translation:
1. IMO, whenever we mention the package then, it should be mentioned as `huggingface_hub`
2. Next, let's make sure that all the URLs work, some URLs are in Hindi and the others in English, do make sure to test this.

happy to re-review after you take a pass.

**Comment by aneeshd27** [comment]
@Wauplin  I have made the necessary changes as mentioned before review.
@Vaibhavs10 I have made the required changes as mentioned during review like replace translated library into English.
It would be great if you take a look at it.
Thank you,
Aneesh.
😄

File: docs/source/hi/_redirects.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+how-to-inference: guides/inference
+how-to-manage: guides/repository
+how-to-model-cards: guides/model-cards
+how-to-upstream: guides/upload
+search-the-hub: guides/search
+guides/manage_spaces: guides/manage-spaces
+package_reference/inference_api: package_reference/inference_client
+
+# Alias for hf-transfer description
+hf_transfer: package_reference/environment_variables#hfhubenablehftransfer
```
Comment (Wauplin) [inline docs/source/hi/_redirects.yml:14]: This is a file used in the English version of the docs to create redirects between old legacy URLs and new ones. We use it when we rename a file for example. So since Hindi documentation is brand new, no need to add this file as no legacy documentation exists.
TL;DR: this file can be removed :)

File: docs/source/hi/_toctree.yml [hunk 5]
```diff
@@ -0,0 +1,79 @@
+- title: "शुरू हो जाओ"
+  sections:
+    - local: index
+      title: (होम) Home
+
```
Comment (Wauplin) [inline docs/source/hi/_toctree.yml:5]:
```suggestion
```

File: docs/source/hi/_toctree.yml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    - local: package_reference/cards
+      title: (रेपो कार्ड और रेपो कार्ड डेटा) Repo Cards and Repo Card Data
+    - local: package_reference/space_runtime
+      title: (स्पेस रनटाइम) Space runtime
+    - local: package_reference/collections
+      title: (संग्रह) Collections
+    - local: package_reference/tensorboard
+      title: (टेंसरबोर्ड logger) TensorBoard logger
+    - local: package_reference/webhooks_server
+      title: (वेबहुक सर्वर) Webhooks server
```
Comment (Wauplin) [inline docs/source/hi/_toctree.yml:79]:
```suggestion
```

I would remove sections that are not translated yet so that users don't have expectations when seen them. It will not brake any existing links/urls as the user will be redirected automatically to the English version in case they click on a link that doesn't exist yet in Hindi. It is not a problem to have only a few links on the sidebar at the beginning for a language that is currently been translated.

I would also remove the guides written in English from this PR.

File: README_hi.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+    <a href="https://pypi.org/project/huggingface-hub"><img alt="downloads" src="https://static.pepy.tech/badge/huggingface_hub/month"></a>
+    <a href="https://codecov.io/gh/huggingface/huggingface_hub"><img alt="Code coverage" src="https://codecov.io/gh/huggingface/huggingface_hub/branch/main/graph/badge.svg?token=RXP95LE2XL"></a>
+</p>
+
+<h4 align="center">
+    <p>
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README.md">English</a> |
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_de.md">Deutsch</a> |
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
+    <p>
```
Comment (Wauplin) [inline README_hi.md:24]: This section adds links to READMEs in other languages. So Hindi should be added to it. Let's keep the same order as in the .yaml files.

```html
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README.md">English</a> |
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_de.md">Deutsch</a> |
        <b>Hindi</b>  | <!--TODO: write Hindi in Hindi -->
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
```

---

Also other READMEs have to be updated to also take into account the Hindi version (they should all reference each other). Especially:
- [README.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README.md?plain=1#L21C1-L23C96)
- [README_de.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README_de.md?plain=1#L21): in this one, can you please add both Hindi and Korean links (we forgot to update it)
- [README_ko.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README_ko.md?plain=1#L21C1-L23C96)

In all 4 READMEs, the order, text and links should be the same. Only the highlighted language (`<b>...</b>`) changes. Could you take care of this? Thanks in advance!

**Review by Wauplin — COMMENTED** [review]
Thanks for making the changes to the workflow files and incorporating feedback from the previous review. I left a few new comments about files that are not necessary yet. We will add them iteratively in future PRs. 

Thank you also for translating the README.md file! That's a very important one to get started. @Vaibhavs10 would you mind taking a look at it?

@aneeshd27 most of my comments are about setting up the translated docs for the first time. I promise that next PRs will be much easier to write :)

File: docs/source/hi/_toctree.yml (hunk 6 above)
Comment (aneeshd27) [reply docs/source/hi/_toctree.yml]: Removed the Hindi translation for the non-translated guides or directories.Is it suggestable that this file be deleted?? As the translation I am doing are based on 3 files that are not mentioned in the _toctree.yml file.

File: README_hi.md (hunk 7 above)
Comment (aneeshd27) [reply README_hi.md]: I have added the required translation links as told in review.I did not add the links for the Hindi Translation as the pull request has not been added to the main branch yet.
@Wauplin Can you please take a look at it?
Thank you,
Aneesh 😄

**Comment by aneeshd27** [comment]
Made Changes as mentioned in Review 2 of the branch aneeshd27-translation

File: README_hi.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+**दस्तावेज़ीकरण**: <a href="https://hf.co/docs/huggingface_hub" target="_blank">https://hf.co/docs/huggingface_hub</a>
+
+**सोर्स कोड**: <a href="https://github.com/huggingface/huggingface_hub" target="_blank">https://github.com/huggingface/huggingface_hub</a>
+
+---
+
+## huggingface_hub लाइब्रेरी में आपका स्वागत है
+
+`huggingface_hub` लाइब्रेरी आपको [हगिंग फेस हब] (https://huggingface.co/) के साथ बातचीत करने की अनुमति देती है, जो रचनाकारों और सहयोगियों के लिए ओपन-सोर्स मशीन लर्निंग का लोकतंत्रीकरण करने वाला एक मंच है। अपनी परियोजनाओं के लिए पूर्व-प्रशिक्षित मॉडल और डेटासेट खोजें या हब पर होस्ट किए गए हजारों मशीन लर्निंग ऐप्स के साथ खेलें। आप समुदाय के साथ अपने स्वयं के मॉडल, डेटासेट और डेमो भी बना और साझा कर सकते हैं। `huggingface_hub` लाइब्रेरी पायथन के साथ इन सभी चीजों को करने का एक आसान तरीका प्रदान करती है।
```
Comment (Vaibhavs10) [inline README_hi.md:38]:
```suggestion
`huggingface_hub` लाइब्रेरी आपको [हगिंग फेस हब](https://huggingface.co/) के साथ बातचीत करने की अनुमति देती है, जो रचनाकारों और सहयोगियों के लिए ओपन-सोर्स मशीन लर्निंग का लोकतंत्रीकरण करने वाला एक मंच है। अपनी परियोजनाओं के लिए पूर्व-प्रशिक्षित मॉडल और डेटासेट खोजें या हब पर होस्ट किए गए हजारों मशीन लर्निंग ऐप्स के साथ खेलें। आप समुदाय के साथ अपने स्वयं के मॉडल, डेटासेट और डेमो भी बना और साझा कर सकते हैं। `huggingface_hub` लाइब्रेरी पायथन के साथ इन सभी चीजों को करने का एक आसान तरीका प्रदान करती है।
```

**Review by Vaibhavs10 — APPROVED** [review]
The README looks good to me! Great job!

File: README_hi.md (hunk 8 above)
Comment (aneeshd27) [reply README_hi.md]: Updated the necessary changes required

**Comment by aneeshd27** [comment]
Updated the necessary changes as mentioned in the review.
@Vaibhavs10 Can you please take a look at it?
Thank you,
Aneesh 😄

File: docs/source/hi/_toctree.yml (hunk 6 above)
Comment (Wauplin) [reply docs/source/hi/_toctree.yml]: @aneeshd27  In my opinion, the doctree should look like this:

```yaml
- title: "Get started"
  sections:
    - local: index
      title: Home
    - local: quick-start
      title: Quickstart
    - local: installation
      title: Installation
```

but translated to Hindi and that's it. No references to the non-translated pages.

File: README_hi.md (hunk 7 above)
Comment (Wauplin) [reply README_hi.md]: > I have added the required translation links as told in review.I did not add the links for the Hindi Translation as the pull request has not been added to the main branch yet.

@aneeshd27  You can already set them to:

```
<a href="https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md">हिंदी</a>
```

in README.md, README_de.md and README_ko.md. So that when we merge this PR, everything will be correct :)

**Comment by aneeshd27** [comment]
I have made the told changes as mentioned in the review.
@Wauplin  Can you please take a look at it?
Thank you,
Aneesh 😄

File: docs/source/hi/concepts/git_vs_http.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
```
Comment (Wauplin) [inline docs/source/hi/concepts/git_vs_http.md:1]: Since this file is not translated and not referenced in the toctree, can you remove it from the PR?
(same for all files in `/concepts` and `/guides`)

File: docs/source/hi/in_translation.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
```
Comment (Wauplin) [inline docs/source/hi/in_translation.md:1]: In the end this file can be removed as it is not referenced anywhere.

**Review by Wauplin — COMMENTED** [review]
Thanks for making the changes @aneeshd27! I think the last thing to change from a technical point of view is to remove all guides/concepts pages from this PR. The goal is that only Hindi-written pages are added in the PR, nothing else.
Thank you in advance!
Wauplin

**Comment by aneeshd27** [comment]
I have made all the changes like deleting files from concepts and guides folder which have not been translated.
@Wauplin Can you please check if the right changes have been made?
Thank you,
Aneesh 😄

**Review by Wauplin — APPROVED** [review]
Perfect! Everything looks good now! Thanks for the additional changes :pray: 

~@Vaibhavs10 would you like to make a last review or are we good to merge?~
**EDIT:** I just realized you already reviewed the README as well. Will merge then :)
